lms/djangoapps/student_account/views.py

```diff
@@ -309,6 +317,22 @@ def _external_auth_intercept(request, mode):
         return external_auth_register(request)
 
 
+def _invalidate_access_token(user, email):
+    """
+    Given a user and email will invalidate all the access tokens associated.
+    """
+    user = User.objects.get(email=email) if not user.is_authenticated() else user
+    clients = Client.objects.all()
+    for client in clients:
```

Comment:
The ACs call for deleting the access tokens: `AccessToken.objects.filter(user=user).delete()`